ci/run_ert_ctests.sh

```diff
@@ -23,12 +23,14 @@ set -x
 build_and_test () {
 	setup
 	build_ert_clib
-	build_ert_dev
 	run_ctest
 }
 
 enable_environment () {
-	source /opt/rh/devtoolset-8/enable
+	if [[ -f /opt/rh/devtoolset-8/enable ]] ; then
+		source /opt/rh/devtoolset-8/enable
+	fi
```

Comment:
Can and should this have been bumped to `devtoolset-9` ? This has been the practice the last years.

Reply:
That would be the only advantage. And even so we should try to get it to run using the newer `gcc-toolset-13` instead once we get to RHEL8.
A sidenote would be that we are quite close to removing the C-code altogether, so this might not be relevant in the too distant future.